proofzero/rollupid
# feat(console): Application transfer to group ownership

### Description



### Related Issues

- Closes #2454 

### Testing

- Create new group
- See app transfer is disabled
- Create new app
- See that app transfer is enabled
- Transfer app
- See that it appears in group app list

### Checklist
